packages/gestalt/src/Masonry/defaultTwoColumnModuleLayout.js

```diff
@@ -40,6 +40,17 @@ function calculateTwoColumnModuleWidth(columnWidth: number, gutter: number): num
   return columnWidth * 2 + gutter;
 }
 
+function calculateSplitIndex(itemsWithoutPositionsLength: number, twoColumnIndex: number): ?number {
+  // If the items length is the same as the batch size we don't set a split index
+  if (itemsWithoutPositionsLength <= TWO_COL_ITEMS_MEASURE_BATCH_SIZE) {
+    return null;
```

Comment:
nit: just return 0 here so return type can always be a number. no need to return null since 0 is also falsey